### ISSUE TITLE
I can’t find the TxID/TxHash of a transaction I made

### ISSUE THREAD

**Issue by aradour** [hirosystems/explorer#1279]
So, 3 months ago I tried to sent my STX balance to Binance from blockchain, the thing is I didn’t look well and I made a big error on sending it without the memo tag. I can still see my transaction blocked in anchor block, and still there. How I can find the TxID and TxHash as Binance has this feature to refund, so I was worried I’ve lost them, but since I still have the address of both when I sent it and the transaction ID is it possible to still solve it out? I’ve lost like $49 and I know now I will put memo tags in feature but I can’t solve it out, I don’t have any history on Binance wallet, only on my blockchain and when I put the ID it’s not accepted idk why, can someone help me? 

#431 #1143 
![IMG_4140](https://github.com/hirosystems/explorer/assets/23318998/f6cd81f7-ddee-4eb9-a69f-a6ecb775eb5a)
![IMG_4142](https://github.com/hirosystems/explorer/assets/23318998/d9672140-47ba-46a8-b8c0-01aac00246a3)

![IMG_4139](https://github.com/hirosystems/explorer/assets/23318998/c58f3d18-dcc8-41b6-a0a1-36e9ae05a174)

**Comment by andresgalante** [comment]
Can you send me the URL where you see your tx? that page should have the information you need

**Comment by aradour** [comment]
The url is the one I open from blockchain, and it leads me to https://explorer.hiro.so/txid/0xdcc10e4ca0409befda74b469d25e78d2d13896f51318c037327e6817e39b2cf6?chain=mainnet 

this should be my transaction that is in anchor I guess if I’m by wrong, is it still possible to refund it on Binance? Because as it shows they need the TxID or TxHash I copied the transaction hash but it says it’s not this, also on Binance I don’t have them on my history ( I read sometimes it can show in the history transaction as non completed but nothing ) I just sent from blockchain to Binance without the memo tag 🙁 i forgot to put it and so it just stays lost, but after I saw they have refund system as it shows, when I put the amount and the code from there and everything it says the TxID and TxHash are wrong. Somehow if can be solved to find where is the code or help me how to fill out the application of binance? It’s a pity because I’ve lost €50 euros around and I hoped to refund them.😕 because I see that they aren’t lost they are still traceable and I see them blocked there, so maybe if I put the right TxID or TxHash I may get them back 🤔 thank you for replying!
